requirements.txt

```diff
@@ -1,6 +1,13 @@
-numpy~=1.21.4
```

Comment:
I need to install `PyQt5-stubs` `types-pkg-resources` `types-setuptools-65.3.0` to be able to run mypy without error. I also see that you have these installed in github action. Should they be included here?